tests/functional/data/docker-leaking-in-layer/foo.conf

```diff
@@ -0,0 +1 @@
+token="ggtt-v-azerty1234"
```

Comment:
This may be overkill, but could we generate this file from `GG_VALID_TOKEN` in  `tests/conftest.py` ?

Reply:
It would avoid having multiple tokens defined in many places.

Reply:
Good idea, doing it.